pymodbus/server/simulator/http_server.py

```diff
@@ -130,15 +130,15 @@ def __init__(
         }
         if custom_actions_module:
             actions_module = importlib.import_module(custom_actions_module)
-            custom_actions_module = actions_module.custom_actions_dict
+            custom_actions_dict = actions_module.custom_actions_dict
```

Comment:
Changing the name is ok, but then you need to make sure the custom_actions_dict is set to None when custom_actions_module is None otherwise line 141 is wrong.